back/spec/lib/spherical_point_impl_active_job_serializer_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe SphericalPointImplActiveJobSerializer do
+  let(:point) { RGeo::Geographic.spherical_factory.point(1.0, 2.0) }
+  let(:serialized_point) { { 'x' => 1.0, 'y' => 2.0 } }
+
+  describe '#serialize' do
+    it 'correctly serializes a RGeo::Geographic::SphericalPointImpl object' do
+      serializer = described_class.send(:new)
+      expect(serializer.serialize(point)).to include(serialized_point)
```

Comment:
`include` or `match`?

Reply:
`serialize` also returns a hash that includes some ActiveJob identifier.